src/components/ui/user-autocomplete.tsx

```diff
@@ -0,0 +1,124 @@
+import { CaretDownIcon, CheckIcon } from "@radix-ui/react-icons";
+import { PopoverClose } from "@radix-ui/react-popover";
+import { useTranslation } from "react-i18next";
+
+import { cn } from "@/lib/utils";
+
+import { Button } from "@/components/ui/button";
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from "@/components/ui/command";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+
+import { Avatar } from "@/components/Common/Avatar";
+
+import { formatName, isUserOnline } from "@/Utils/utils";
+import { UserBase } from "@/types/user/user";
+
+interface UserCompleteProps {
+  id: string;
+  disabled?: boolean;
+  value: UserBase | undefined;
+  isLoading?: boolean;
+  onSelect: (user: UserBase | null) => void;
+  options: {
+    users: UserBase[];
+  };
```

Comment:
why nest inside an object?

```suggestion
  options: UserBase[];
```